include/boost/math/tools/header_deprecated.hpp

```diff
@@ -8,10 +8,10 @@
 
 #ifdef _MSC_VER
 // Expands to "This header is deprecated; use expr instead."
-#define BOOST_MATH_HEADER_DEPRECATED(expr) #prama message("This header is deprecated; use " expr " instead.")
```

Comment:
Shouldn't the fix have been `#pragma` to remove the `#prama` typo?

Reply:
That may have worked. In the GCC case the spelling was right but the `#warning` did not expand correctly so I used `_Pragma` instead